src/formatDate.js

```diff
@@ -50,7 +50,57 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const SEPARATOR_INDEX = 3;
+  const START_ITERATION = 0;
+  const END_ITERATION = 3;
+  const LAST_DIGITS = 30;
+  const CURRENT_CENTURY_YEAR = '20';
+  const PREVIOUS_CENTURY_YEAR = '19';
+  const stringItems = date.split(fromFormat[SEPARATOR_INDEX]);
+  const newDate = [];
+
+  for (let i = START_ITERATION; i < END_ITERATION; i++) {
+    let toFormatItem = toFormat[i];
+
+    switch (true) {
```

Comment:
currently you have only one case to this switch, so don't make any sense to use switch case here
you can use switch case statement only when you have a multiple if else blocks